CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# dbt_fivetran_log v1.8.0
+[PR #130](https://github.com/fivetran/dbt_fivetran_log/pull/130) includes the following updates:
+
+## 🚨 Breaking Changes 🚨
+> ⚠️ Since the following changes result in the table format changing, we recommend running a `--full-refresh` after upgrading to this version to avoid possible incremental failures.
+- For Databricks All Purpose clusters the `fivetran_platform__audit_table` model will now be materialized using the delta table format (previously parquet). 
+  - Delta tables are generally more performant than parquet and are also more widely available for Databricks users. Previously, the parquet file format was causing compilation issues on customers managed tables.
+
+## Documentation Updates
+- Updated the `sync_start` and `sync_end` field descriptions for the `fivetran_platform__audit_table` to explicitly define that these fields only represent the sync start/end times for when the connector wrote new or modified existing records to the specified table.
+
+## Under the Hood
```

Comment:
Are the consistency and integrity tests under the hood? Or should we not call them out?

Reply:
Yeah they are under the hood - would be good to include in the CHANGELOG. Will update!

Reply:
Added

Reply:
This was added under Documentation Updates. Make sure to move it down!